src/abi.h

```diff
@@ -327,4 +327,43 @@ struct netplan_net_definition {
     /* netplan-feature: eswitch-mode */
     char* embedded_switch_mode;
     gboolean sriov_delay_virtual_functions_rebind;
+
+    /* vxlan */
+    guint vxlan_vni;
+    NetplanNetDefinition* vxlan_link;
+    char* vxlan_bridge;
+    GArray* vxlans;
+    gboolean neigh_suppress;
+
+    /* vrf */
+    char* vrf;
+    guint vrf_table;
+    NetplanNetDefinition* vrf_link;
+
+    struct {
+        char* remote;
+        char* local;
+        char* group;
+        guint tos;
+        guint ttl;
+        gboolean mac_learning;
+        guint fdb_ageing;
+        guint max_fdb_entries;
+        gboolean reduce_arp_proxy;
+        gboolean l2_miss_notification;
+        gboolean l3_miss_notification;
+        gboolean route_short_circuit;
+        gboolean udp_checksum;
+        gboolean udp6_zero_checksum_tx;
+        gboolean udp6_zero_checksum_rx;
+        gboolean remote_checksum_tx;
+        gboolean remote_checksum_rx;
+        gboolean group_policy_extension;
+        gboolean generic_protocol_extension;
+        guint destination_port;
+        char* port_range;
+        guint flow_label;
+        gboolean ip_do_not_fragment;
+        gboolean independent;
+    } vxlan_params;
```

Comment:
Thank you for moving any new fields in this `netplan_net_definition` struct to the very end, this way we'll only append the ABI and not break it :-)

Reply:
No problem!